gensim/summarization/pagerank_weighted.py

```diff
@@ -60,24 +62,28 @@ def pagerank_weighted(graph, damping=0.85):
         Nodes of `graph` as keys, its ranks as values.
 
     """
-    adjacency_matrix = build_adjacency_matrix(graph)
-    probability_matrix = build_probability_matrix(graph)
+    coeff_adjacency_matrix = build_adjacency_matrix(graph, coeff=damping)
+    probabilities = (1 - damping) / float(len(graph))
 
-    pagerank_matrix = damping * adjacency_matrix.todense() + (1 - damping) * probability_matrix
+    pagerank_matrix = coeff_adjacency_matrix.toarray()
```

Comment:
Even for sparse graph we need O(len(graph)**2) memory =(
For example, graph with 50000 nodes it will be needed 50000*50000 *4 bytes = 9536 MiB.
That's not cool.

Reply:
but why? matrix has a little number of edges.

Reply:
-1 pass by array on len(edges) (and actually by matrix on `(len(graph), len(graph)` due to  transforming from sparse to dense)

Reply:
Because to adjacency_matrix are added probability of damping. So matrix becomes not sparse. I'm gonna try to find the other ways to calculate pagerank but in other PR I think.